### ISSUE TITLE
http://api.bcdevexchange.org Footer missing links

### ISSUE THREAD

**Issue by kelpisland** [BCDevExchange/BCDevExchange-Site#18]
Footer is missing links to Privacy, Accessibility, Copyright

![screen shot 2016-08-04 at 4 37 19 pm](https://cloud.githubusercontent.com/assets/9368818/17421909/46160a5e-5a62-11e6-9b7c-a7f23785de73.png)
![screen shot 2016-08-04 at 4 41 33 pm](https://cloud.githubusercontent.com/assets/9368818/17421912/537b9c86-5a62-11e6-9a46-c041ca8ba181.png)

**Comment by ccoldwell** [comment]
they all seem to be there now

**Comment by kelpisland** [comment]
[API List](http://api.bcdevexchange.org) is part of the "mini-aps" I think... still only has "disclaimer" but I'm looking at the PROD version, is there a dev site I can't see?

**Comment by ccoldwell** [comment]
not sure what you are seeing, or what you are talking about

**Comment by kelpisland** [comment]
OK - this is not part of the re-platform so let's just close this one for now.
